fuel/app/classes/controller/questions.php

```diff
@@ -31,4 +31,28 @@ public function action_import()
 
 		return Response::forge($theme->render());
 	}
+
+	// this exists as an alternative to the above for use with the React component
+	// once that is confirmed and is ready for real use as a replacement for the current question importer,
+	//  remove the above action and rename this one 'get_import', remove extra routes etc.
+	public function get_import2()
```

Comment:
Once the new question importer has been checked out and we've confirmed we want to stick with it, another commit to remove the old code and consolidate routes etc. will be necessary